soccer/src/soccer/strategy/agent/position/defense.cpp

```diff
@@ -164,6 +201,22 @@ std::optional<RobotIntent> Defense::state_to_task(RobotIntent intent) {
     } else if (current_state_ == MARKING) {
         // Marker marker = Marker((u_int8_t) robot_id_);
         return marker_.get_task(intent, last_world_state_, this->field_dimensions_);
+    } else if (current_state_ == STEALING) {  // wall stealer
+        auto collect_cmd = planning::MotionCommand{"collect"};
+        intent.motion_command = collect_cmd;
+        intent.dribbler_speed = 255.0;
+        return intent;
+    } else if (current_state_ == CLEARING) {
+        planning::LinearMotionInstant target{clear_point_};
```

Comment:
can we add a rotate state here, with kicking at end? you probably need to merge in ros2